rest-server/core/src/main/scala/com/actionml/core/spark/SparkContextSupport.scala

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright ActionML, LLC under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * ActionML licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.actionml.core.spark
+
+import cats.data.Validated
+import cats.data.Validated.{Invalid, Valid}
+import com.actionml.core.validate._
+import com.mongodb.spark.MongoSpark
+import com.mongodb.spark.rdd.MongoRDD
+import com.typesafe.scalalogging.LazyLogging
+import org.apache.spark.{SparkConf, SparkContext}
+import org.bson.Document
+import org.json4s._
+import org.json4s.jackson.JsonMethods._
+
+
+trait SparkContextSupport extends LazyLogging {
+  import com.actionml.core.spark.SparkContextSupport._
+
+  def createRdd(sc: SparkContext): MongoRDD[Document] = {
+    MongoSpark.load[Document](sc)
+  }
+
```

Comment:
this method looks like is should be in the mongo client, maybe an extension of the base client that supports Spark. leaving the base Mongo client free of the need for Spark dependencies.

There will be a similar requirement for Elasticsearch, it must write an RDD (or Dataframe) to ES but should not pollute the SparkContextSupport with ES dependencies. Again, maybe classes for `ESClient` and `SparkESClient extends ESClient` would be the way to go.